data_encryption/readme/ROADMAP.rst

```diff
@@ -0,0 +1,5 @@
+For now the encryption is dependent on the environment. It has been designed
+to store the same kind of data with different values depending on the environement
+(dev, preprod, prod...).
+An improvement could be to split this in 2 modules. But the environment stuff
+is not a big constraint.
```

Comment:
Question: what happens if you grab data from PROD into your DEV env? Data will be unreadable, right?
How would you deal w/ it?

Reply:
I am not sure I understand your question.
To resume (I'll mention this in the configure part of the readme) you have one "encryption key" by environment.
So, in your dev environment, if you try to read a prod data, it will fail and raise an error, because the prod encryption key won't be present. But, it should not be possible.
I mean, this module is really low level and is meant to be used by other (example with server_environment_data_encryption) but these module should not let a dev environment even try to read a prod data... It should not be possible, and if it is, it will raise an error anyway.

The idea would be to configure all encryption key in your prod environment, and only the environment key for others.
For instance, in your dev config file, you will have 
encryption_key_dev = XXX
running_env=dev

In your prod environment, you will have : 
encryption_key_dev = XXX
encryption_key_prod=YYY
running_env=prod

This allows you to set the values for all your environment directly from the production. (there is a was to change environment to set read the values in server_environment_data_encryption)
And indeed, in your dev environment, Odoo won't be able to decrypt the data.
But in principle, no module should make it possible.

Not sure I answered you, do not hesitate to precise your question if I was not clear!

Reply:
ok, thanks for the explanation and the README update :)